src/API/Authentication.php

```diff
@@ -309,20 +309,27 @@ public function codeExchange($code, $redirectUri)
      * @param string $username
      * @param string $password
      * @param string $realm
-     * @param array $options {
-     *
-     *      @var string $scope    Optional
-     *      @var string $audience Optional.
-     * }
+     * @param string $scope
+     * @param string $audience
      *
      * @return array
      */
-    public function login($username, $password, $realm, array $options = [])
+    public function login($username, $password, $realm, $scope = null, $audience = null)
```

Comment:
> should we default here to scope ="openid"

no, it does not make a lot of sense in this endpoint 

>  if we should retain options as a just in case

I dont think so, we can add later if we need